rakelib/kokoro/kokoro.rb

```diff
@@ -143,7 +164,7 @@ def check_links location_list, base, tail
         puts err
       end
       checked_links = out.split "\n"
-      checked_links.select! { |link| link =~ /\[\d+\]/ && !link.include?("[200]") }
+      checked_links.select! { |link| link =~ /^\[\d+\]/ && !link.include?("[200]") }
```

Comment:
Is the "^" intentional?

Reply:
@TheRoyalTnetennba Yeah. It's because the output is currently being duplicated: linkinator prints out each result while running, and then at the end it prints it out again, indented underneath the page that the link was from. So I modified the filter to take only the non-indented output.